H7A3Zi_AzureRTOS_ThreadX/Core/Src/app_threadx.c

```diff
@@ -34,7 +34,8 @@
 
 /* Private define ------------------------------------------------------------*/
 /* USER CODE BEGIN PD */
-#define THREAD_STACK_SIZE	1024
+#define TX_APP_MEM_POOL_SIZE	3*1024
```

Comment:
why we used 3KB ?

Reply:
Because we are creating 2 threads, and we are allocating 1 kb to each thread so 3 kb of pool will be sufficient enough.